### PR TITLE
added team member test

### PR DESCRIPTION
**Pull request by acostacos**
Tests do not pass due to the following issue: https://github.com/gambitph/Stackable/issues/1074

### CODE REVIEW

File: cypress/integration/v2/blocks/team-member.spec.js [hunk 1]
```diff
@@ -57,6 +61,241 @@ function switchDesign() {
...
+	assertAligns( 'Align', '.ugb-inner-block', { viewport } )
+
+	// Container Tab
+	cy.collapse( 'Container' )
+
+	assertContainer( '.ugb-team-member__item', { viewport }, 'column%sBackgroundMediaUrl' )
+
+	range( 1, 4 ).forEach( idx => {
+		cy.setBlockAttribute( {
+			[ `image${ idx }Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
```
Comment (legaspielaine) [inline cypress/integration/v2/blocks/team-member.spec.js:87]: image3Url is also set for tablet & mobile even if the columns set for tablet/mobile is only 2 (default). We can move the cy.adjust( 'Columns', 3 ) outside of desktopOnly.

File: cypress/support/stackable/commands/controls.js [hunk 2]
```diff
@@ -64,6 +64,15 @@ Cypress.Commands.overwrite( 'adjust', ( originalFn, ...args ) => {
...
 	}
+	if ( label === 'Social Buttons' ) {
+		args.shift()
+		const socialButtons = args[ 0 ]
+		Object.keys( socialButtons ).forEach( key => {
+			optionsToPass.isNestledToggle = true
+			cy.toggleControl( key, socialButtons[ key ], optionsToPass )
+		} )
+		return cy.get( '.block-editor-block-list__block.is-selected' )
+	}
```
Comment (blankart) [inline cypress/support/stackable/commands/controls.js:75]:
```suggestion
```

File: cypress/support/gutenberg/commands/controls.js [hunk 3]
```diff
@@ -181,9 +181,12 @@ function toggleControl( name, value, options = {} ) {
 	const {
 		isInPopover = false,
 		beforeAdjust = () => {},
+		isNestledToggle = false,
```
Comment (blankart) [inline cypress/support/gutenberg/commands/controls.js:184]:
```suggestion
```

File: cypress/support/gutenberg/commands/controls.js [hunk 4]
```diff
@@ -181,9 +181,12 @@ function toggleControl( name, value, options = {} ) {
...
 	} = options
 
-	const selector = () => cy.getBaseControl( name, { isInPopover } )
+	const selectorOptions = ! isNestledToggle ? { isInPopover } : { isInPopover, customParentSelector: '.components-base-control' }
+
+	const selector = () => cy.getBaseControl( name, selectorOptions )
```
Comment (blankart) [inline cypress/support/gutenberg/commands/controls.js:189]:
```suggestion
	const selector = () => cy.getBaseControl( name, { isInPopover, customParentSelector: '' } )
```

**Review by blankart — APPROVED** [review]

**Review by blankart — APPROVED** [review]

**Pull request merged**